File: C#/Console Applications/15.cs
﻿using System;
int a, b;
System.Console.WriteLine("Enter a value:");
a = Convert.ToInt32(System.Console.ReadLine());
System.Console.WriteLine("Enter b value:");
b = Convert.ToInt32(System.Console.ReadLine());
System.Console.WriteLine(" Values before swapping: a = " + a + " b = " + b);
a = a ^ b;
b = a ^ b;
a = a ^ b;
System.Console.WriteLine(" Values after swapping: a = " + a + " b = " + b);

File: VB.Net/15.vb
Module Module1

    Sub Main()
        Dim a As Integer
        Dim b As Integer
        System.Console.WriteLine("Enter a value:")
        a = Convert.ToInt32(Console.ReadLine())
        System.Console.WriteLine("Enter b value:")
        b = Convert.ToInt32(Console.ReadLine())
        System.Console.WriteLine("Values before swap a=" & a & "b=" & b)
        a = a + b
        b = a - b
        a = a - b
        System.Console.WriteLine("Values after swap a=" & a & "b=" & b)
    End Sub

End Module
